agent/agent.go

```diff
@@ -97,19 +97,27 @@ func (a *Agent) Start() {
 		panic(err)
 	}
 
-	pipeline := mappings.NewFlowMappingPipeline([]mappings.FlowEnhancer{gfe})
+	ns := tprobes.NewNetNSProbe(a.Graph, root)
+	nl := tprobes.NewNetLinkProbe(a.Graph, root)
+	ovs := tprobes.NewOvsdbProbe(a.Graph, root, ovsmon)
+	flowEnhancers := []mappings.FlowEnhancer{gfe}
+
+	if _, err := config.GetConfig().GetSection("docker"); err == nil {
+		dockerEnhancer, err := mappings.NewDockerMapper(ns)
```

Comment:
a call like mappings.NewDockerMapperFromConfig() will be more consistant in our code